### PR TITLE
Merge snapshot options

### PR DESCRIPTION
**Pull request by dureuill**
No API change

---

# Summary

The `--schedule-snapshot` and `--snapshot-interval-sec` options were merged in https://github.com/meilisearch/meilisearch/issues/3131

---

# Changes

- Remove any mention of `--snapshot-interval-sec`
- Propagate changes to telemetry
- Change specification of `--schedule-snapshot` to reflect its new behavior

# Out Of Scope

N/A 

---

# Attention To Reviewers

`--schedule-snapshot` is the first option that is "tri-valued" (absent/flag only/flag with a value). I tried to be clear but the wording should be reviewed carefully

---

## Misc

- Update OpenAPI specification file _(if needed; Apply the `OpenApi` label)_ N/A
- [x] Update telemetry datapoints _(if needed; Apply the `Telemetry` label)_

### CODE REVIEW

File: text/0119-instance-options.md [hunk 1]
```diff
@@ -318,7 +311,7 @@ This command will throw an error if:
 
 This behavior can be modified with the `--ignore-snapshot-if-db-exists` and `--ignore-missing-snapshot` options, respectively.
 
-#### 3.3.18. Ignore missing snapshot
+#### 3.3.16. Ignore missing snapshot
```
Comment (gmourier) [inline text/0119-instance-options.md:314]:
```suggestion
#### 3.3.17. Ignore missing snapshot
```

**Review by gmourier — CHANGES_REQUESTED** [review]
I've asked for a review from @maryamsulemani97 regarding the technical explanations of the "tri-valued" concept you explaining lines 285-287. It seems good to me.

**Review by maryamsulemani97 — APPROVED** [review]
LGTM! 🤖

File: text/0119-instance-options.md (hunk 1 above)
Comment (gmourier) [inline text/0119-instance-options.md:314]:
```suggestion
#### 3.3.17. Ignore missing snapshot
```

**Pull request merged**